python/dask_cudf/dask_cudf/batcher_sortnet.py

```diff
@@ -3,8 +3,17 @@
 Adapted from https://en.wikipedia.org/wiki/Batcher_odd%E2%80%93even_mergesort
 """
 import math
+from operator import getitem
+
+import numpy as np
+import toolz
```

Comment:
Not critical, but it might be nice at some point to optionally use `cytoolz` if it is available and otherwise use `toolz` as a fallback.